app/views/_pod-template.html

```diff
@@ -7,194 +7,67 @@
-            <span class="hash" title="{{imagesByDockerReference[container.image].metadata.name}}">{{imagesByDockerReference[container.image].metadata.name | stripSHAPrefix | limitTo: 7}}</span>
-            <span ng-if="imagesByDockerReference[container.image].dockerImageMetadata.Size" class="small text-muted nowrap">
-              {{imagesByDockerReference[container.image].dockerImageMetadata.Size | humanizeSize}}
-            </span>
-          </span>
-        </div>
-      </div>
-      <div ng-if="imagesByDockerReference && container.image && (image = imagesByDockerReference[container.image])" class="pod-template-build">
-        <div row class="icon-row" ng-if="build = (image | buildForImage : builds)">
-          <div class="icon-wrap">
-            <span class="fa fa-refresh" aria-hidden="true"></span>
-          </div>
-          <div flex class="word-break">
-            <span class="pod-template-key">Build:</span>
-            <span ng-if="build | configURLForResource">
-              <a ng-href="{{build | configURLForResource}}">{{build | buildConfigForBuild}}</a>,
-            </span>
-            <a ng-href="{{build | navigateResourceURL}}">
-              <span ng-if="(build | annotation : 'buildNumber')">#{{build | annotation : 'buildNumber'}}</span>
-              <span ng-if="!(build | annotation : 'buildNumber')">{{build.metadata.name}}</span>
-            </a>
-          </div>
-        </div>
-        <div row class="icon-row" ng-if="build.spec.source">
-          <div class="icon-wrap">
-            <span class="fa fa-code" aria-hidden="true"></span>
-          </div>
-          <div flex class="word-break">
-            <span class="pod-template-key">Source:</span>
-            <span ng-switch="build.spec.source.type">
-              <span ng-switch-when="Git">
-                <span ng-if="build.spec.revision.git.commit">
-                  {{build.spec.revision.git.message}}
-                  <osc-git-link
-                    class="hash"
-                    uri="build.spec.source.git.uri"
-                    ref="build.spec.revision.git.commit">{{build.spec.revision.git.commit | limitTo:7}}</osc-git-link>
-                  <span ng-if="detailed && build.spec.revision.git.author">
-                    authored by {{build.spec.revision.git.author.name}}
-                  </span>
-                </span>
-                <span ng-if="!build.spec.revision.git.commit">
-                  <osc-git-link uri="build.spec.source.git.uri">{{build.spec.source.git.uri}}</osc-git-link>
-                </span>
-              </span>
-              <span ng-switch-default>
-                {{build.spec.source.type || 'Unknown'}}
-              </span>
-            </span>
-          </div>
-        </div>
-      </div>
+  <div class="pod-template-container">
+    <div class="pod-template-block" ng-repeat="container in initContainers | orderBy:'name'">
```

Comment:
Are `status.containers` coming back in a different order than `spec.containers` in the pod JSON?